base.yml

```diff
@@ -0,0 +1,58 @@
+copyright: Copyright &copy; Numtide
+
+extra:
+  social:
+    - icon: fontawesome/brands/github
+      link: https://github.com/numtide
+      name: Numtide on Github
+    - icon: fontawesome/brands/twitter
+      link: https://twitter.com/numtide
+      name: Numtide on Twitter
+  generator: false # hide the line "Made with Material for MkDocs"
+
+extra_css:
+  - stylesheets/extra.css
```

Comment:
Turns out that these settings are relative to the docs/ folder, so the approach I'm taking doesn't work. @zimboflyman I think the repo will have to be converted to a template

Reply:
For now I'm pointing this to the published assets URL and that seems to work